src/test/java/com/rultor/agents/github/qtn/QnAskedByTest.java

```diff
@@ -78,4 +79,40 @@ public void excludesRultorFromListOfCommanders() throws Exception {
             )
         );
     }
+
+    /**
+     * QnAskedBy can include all architects in the list of commanders.
+     * @throws Exception In case of error.
+     */
+    @Test
+    public void includesArchitectsInListOfCommanders() throws Exception {
+        final MkGithub github = new MkGithub();
+        final Repo repo = github.randomRepo();
+        repo.collaborators().add("testuser2");
+        final Issue issue = repo.issues().create("needs", "deployment");
+        final String action = "deploy";
+        issue.comments().post(action);
+        final Comment.Smart comment = new Comment.Smart(
+            issue.comments().get(1)
+        );
+        final Question question = Mockito.mock(Question.class);
+        final QnAskedBy qab = new QnAskedBy(
+            new Profile.Fixed(
+                new XMLDocument(
+                    String.format(
+                        "<p><entry key='architect'>%s</entry></p>",
+                        repo.github().users().self().login()
+                    )
+                )
+            ),
+            String.format(
+                "/p/entry[@key='%s']/entry[@key='commanders']/item/text()",
```

Comment:
@pinaf jup :)  without the change you only get the commanders from :
```
          logins.addAll(new Crew(repo).names());
         logins.addAll(xml.xpath(this.xpath));
```
and hence the auth check fails because the user is in neither list.
It's the exact situation my account here faces right now reproduced: 
I'm not to be found under the above xpath, neither am I a collaborator on the repo => test fails.